src/txorphanage.cpp

```diff
@@ -246,6 +246,9 @@ bool TxOrphanage::HaveTxToReconsider(NodeId peer)
 
 void TxOrphanage::EraseForBlock(const CBlock& block)
 {
+    // No orphans, nothing to do
```

Comment:
I find the comment redundant, it states exactly what the code already states clearly, doesn't add any new info